### PR TITLE
docs: Add note on webpack-watch resource consumption

### PR DESCRIPTION
**Pull request by sbvkrishna**
### Problem

Note on Webpack watch resource consumption.

### Areas of Impact

README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,12 +142,14 @@ For that, you will need to modify the `docker-compose.yml` file to mount the `sr
 
 For example:
 ```
-service:
+services:
   bookbrainz-site:
     command: npm run debug
     volumes:
+      - "./config/config.json:/home/bookbrainz/bookbrainz-site/config/config.json:ro"
```
Comment (sbvkrishna) [inline README.md]: Thought this will help prevent replacing *config* line with *src* line

Comment (MonkeyDo) [reply README.md]: Good thinking !

**Comment by MonkeyDo** [comment]
Thanks for adding that, it's a useful note !
I don't have the point of view of starting on the project for the first time, so the issues you run into and your point of view are very useful for other starting contributors.

I'll be making more modifications to the repo documentation, and explain things further.
I might ask for you to review it to see if it's all understandable, if that's OK. :)

**Comment by sbvkrishna** [comment]
Sure. It would be a pleasure!🙃

**Pull request merged**